models/user_context.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "github.com/google/uuid"
+
+type UserContext struct {
+	Email          string
+	UserID         uuid.UUID `json:"sub"`
```

Comment:
Ensure that JSON tags in the `UserContext` struct match the expected JWT claim keys, and consider renaming them for consistency if necessary. [medium]